File: app/src/main/java/com/qxy/siegelions/CustomApplication.java
package com.qxy.siegelions;

import android.app.Application;

import com.bytedance.sdk.open.douyin.DouYinOpenApiFactory;
import com.bytedance.sdk.open.douyin.DouYinOpenConfig;

/**
 * 自定义{@link Application}
 * @author MadSamurai
 */
public class CustomApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        String clientkey = "awobitbo8w4mf83r"; // 需要到开发者网站申请并替换
        DouYinOpenApiFactory.init(new DouYinOpenConfig(clientkey));
    }
}

File: app/src/main/java/com/qxy/siegelions/CustomApplication.kt
package com.qxy.siegelions

import android.app.Application
import com.bytedance.sdk.open.douyin.DouYinOpenApiFactory
import com.bytedance.sdk.open.douyin.DouYinOpenConfig
import com.qxy.siegelions.util.CLIENT_KEY

class CustomApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        val clientkey = CLIENT_KEY // 需要到开发者网站申请并替换
        DouYinOpenApiFactory.init(DouYinOpenConfig(clientkey))
    }
}